.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: File a bug report
+title: Describe the problem
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Your issue will be triaged by the RNW team according to this process: https://github.com/microsoft/react-native-windows/wiki/Triage-Process
+-->
+## Environment
+Run the following in your terminal and copy the results here.
+1. `react-native -v`:
+2. `react-native run-windows --info`:
+3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
+
+<!-- Consider including this information as well:
+What SDK version are you building for? Choose from 10.0.15063, 10.0.16299, 10.0.18362, etc.
+- Target Platform Version(s):
+What device(s) are you targeting? Choose any from Desktop, Xbox, Hololens)
+- Target Device(s):
+Which version of Visual Studio are you using? Specify Visual Studio 2017 or Visual Studio 2019
+- Visual Studio Version:
+Which build configuration are you running? Choose from Debug, DebugBundle, Release, ReleaseBundle 
+- Build Configuration:
+-->
+
+## Steps To Reproduce
+Provide a detailed list of steps that reproduce the issue.
+
+1.
+2.
+
+## Expected Results
+Describe what you expected to happen.
+
+<!--
+Troubleshooting
+  If you see build failure on `react-native run-windows`, please try again with 'react-native run-windows --logging' and provide the output.
```

Comment:
This was an odd callout. Should we have more troubleshooting items?

Reply:
@chrisglein  `--logging` is a good first step. The next step if running into build issues is to open the generated .binlog file in the MSBuild Structured Log Viewer and navigate through the failure to glean some insight into what went wrong